usim/py/resources/container.py

```diff
@@ -0,0 +1,110 @@
+from .base import BaseResource, Put, Get
+
+
+class ContainerPut(Put):
+    """Request to put ``amount`` of resources into the ``container``"""
+    def __init__(self, container: 'Container', amount: float):
+        if amount <= 0:
+            raise ValueError('amount must be greater than 0')
+        self.amount = amount
+        super(ContainerPut, self).__init__(container)
+
+
+class ContainerGet(Get):
+    """Request to get ``amount`` of resources out of the ``container``"""
+    def __init__(self, container, amount):
+        if amount <= 0:
+            raise ValueError('amount must be greater than 0')
+        self.amount = amount
+        super(ContainerGet, self).__init__(container)
+
+
+class Container(BaseResource):
+    """
+    Resource with ``capacity`` of continuous, indistinguishable content
```

Comment:
The class implements `level` to get information about the current `capacity`. Should this description better be reformulated to reflect this property?

Reply:
The docstring now refers to the level instead of a vague "current amount".

Reply:
The docstring now refers to the level instead of a vague "current amount".